### ISSUE TITLE
HTML data table not updating after new row added via post

### ISSUE THREAD

**Issue by gedw99** [soypat/htmxgo-template#2]
On chrome. Not sure yet if its just not implemented or is chrome issue.

I am guessing its just not implemented yet.

**Comment by gedw99** [comment]
Hey @soypat 

I am also working on a htmx that is NOT JS based..

**Comment by soypat** [comment]
Yeah I am aware of this one, gotta refresh. Unsure how to trigger an htmx event from hyperscript to do this...

**Comment by gedw99** [comment]
Have a look at this one:

https://github.com/joerdav/go-htmx-examples

it has what your looking for ! It helped me a ton :) 

it’s from : 

https://github.com/a-h/templ/network/dependents

Templ is a golang templated compiker . A lot of devs are using it with htmx.

**Comment by gedw99** [comment]
https://github.com/acaloiaro/hugo-htmx-go-template Looks pretty damn cool too

**Comment by soypat** [comment]
Woah, those are nice- thanks @gedw99 !
